### PR TITLE
Rename 'new' and 'old' fields in operation and all related refactoring

### PR DESCRIPTION
**Pull request by vshcherb**

### CODE REVIEW

**Comment by vshcherb** [comment]
1. No changes in admin.html
2. json doesn't support comments! // TODO write validation rules for create/delete/edit
And this validation will be anyway in code not here

File: opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ private String getServerUser(HttpSession session) {
     		if(loginObj == null) {
     			throw new IllegalArgumentException("There is nothing to edit cause signup obj doesn't exist");
     		} else {
-    			op.addOld(loginObj.getParentHash(), 0);
```
Comment (vshcherb) [inline opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java:179]: ????

File: opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java [hunk 2]
```diff
@@ -286,7 +286,7 @@ private String getServerUser(HttpSession session) {
     				throw new IllegalArgumentException("There is nothing to edit cause login obj doesn't exist");
     			}
     		} else {
-    			op.addOld(loginObj.getParentHash(), 0);
```
Comment (vshcherb) [inline opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java:289]: ????

File: opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpBlockchainRules.java [hunk 3]
```diff
@@ -295,8 +300,12 @@ private boolean validateRule(OpBlockChain blockchain, OpObject rule, OpOperation
 		for(String key : refObjsCache.keySet()) {
 			((JsonObject) refsMap.get(key)).addProperty(OpOperation.F_TYPE, refObjsCache.get(key).getParentType());
 		}
-		EvaluationContext ctx = new EvaluationContext(blockchain, formatter.toJsonElement(o).getAsJsonObject(),
-				deletedArray, refsMap);
+		JsonObject opJsonObj = formatter.toJsonElement(o).getAsJsonObject();
+		JsonElement createdElement = opJsonObj.remove(F_CREATE);
```
Comment (vshcherb) [inline opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpBlockchainRules.java]: remove????

File: opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpOperation.java [hunk 4]
```diff
@@ -106,28 +106,28 @@ public String getRawHash() {
...
 	
-	public List<String> getOld() {
-		return getStringList(F_OLD);
+	public List<String> getDeleted() {
+		return getStringList(F_DELETE);
 	}
 	
-	public void addOld(String hash, int ind) {
-		addOrSetStringValue(F_OLD, hash + ":" + ind);
+	public void addCreated(String hash, int ind) {
```
Comment (vshcherb) [inline opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpOperation.java]: ????????

File: opendb-api/src/main/resources/admin.html [hunk 5]
```diff
@@ -53,665 +52,504 @@ <h3>Admin actions</h3>
...
+        $("#objects-type-list").change(function() {
+            loadObjectsData();
+        });
 
-    $("#block-bootstrap-btn").click(function(){
-      $.post("/api/mgmt/bootstrap", {})
-        .done(function(data){  $("#result").html(data); loadData(); })
-        .fail(function(xhr, status, error){  $("#result").html("ERROR: " + error); loadData(); });
-    });
+        $("#block-create-btn").click(function(){
```
Comment (vshcherb) [inline opendb-api/src/main/resources/admin.html]: ??

File: opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpBlockchainRules.java (hunk 3 above)
Comment (upers) [reply opendb-core/src/main/java/org/openplacereviews/opendb/ops/OpBlockchainRules.java]: Yes. Validation doesn't need "create" field

File: opendb-api/src/main/resources/admin.html (hunk 5 above)
Comment (upers) [reply opendb-api/src/main/resources/admin.html]: old -> delete
new -> create

File: opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java (hunk 1 above)
Comment (upers) [reply opendb-core/src/main/java/org/openplacereviews/opendb/api/OpApiController.java:179]: Fixed: https://github.com/OpenPlaceReviews/opendb/pull/10/commits/f0fb01f5e5496fa3f743e5560f3620297c1950ed

**Pull request merged**